packages/blockchain-wallet-v4-frontend/src/data/components/identityVerification/model.ts

```diff
@@ -1,22 +1,29 @@
 import { compose, includes, propOr } from 'ramda'
 
+import { WalletFormType } from '../../../../../../typings/redux-form'
```

Comment:
can't we use absolute imports here?

Reply:
will check it, in general that should be way to go since I also do hate such relative imports :D

Reply:
done